addons/godot-vim/godot-vim.gd

```diff
@@ -94,6 +94,7 @@ var the_key_map : Array[Dictionary] = [
     { "keys": ["C"],                            "type": OPERATOR, "operator": "change" },
     { "keys": ["Shift+C"],                      "type": OPERATOR_MOTION, "operator": "change", "motion": "move_to_end_of_line", "motion_args": { "inclusive": true } },
     { "keys": ["X"],                            "type": OPERATOR_MOTION, "operator": "delete", "motion": "move_by_characters", "motion_args": { "forward": true, "one_line": true }, "context": Context.NORMAL },
+    { "keys": ["S"],                            "type": OPERATOR_MOTION, "operator": "delete_and_enter_insert_mode", "motion": "move_by_characters", "motion_args": { "forward": true, "one_line": true }, "action_args": { "insert_at": "inplace" }, "context": Context.NORMAL },
```

Comment:
This is a one-line, you can remove the `delete_and_enter_insert_mode` function:
```suggestion
	{ "keys": ["S"],                            "type": OPERATOR_MOTION, "operator": "change", "motion": "move_by_characters", "motion_args": { "forward": true }, "context": Context.NORMAL },
```

Reply:
oh cool! addressed. Tested and it looks good